modules/swagger-codegen/src/main/resources/csharp/api.mustache

```diff
@@ -297,12 +297,12 @@ namespace {{packageName}}.{{apiPackage}}
 
             {{#returnType}}
             return new ApiResponse<{{{returnType}}}>(localVarStatusCode,
-                localVarResponse.Headers.ToDictionary(x => x.{{^netStandard}}Name{{/netStandard}}{{#netStandard}}Key{{/netStandard}}, x => x.Value.ToString()),
+                localVarResponse.Headers.ToDictionary(x => x.{{^netStandard}}Name{{/netStandard}}{{#netStandard}}Key{{/netStandard}}, x => x.Value.ToArray()),
```

Comment:
iirc, this `ToArray()` won't work in .NET 3.5/4.x. Can you regenerate samples and test both the `^netStandard` and `#netStandard` branches here?

Reply:
Thank you for your quick response!

I will take a look at the earlier .NET versions.